rootfs/runner/init

```diff
@@ -47,9 +48,8 @@ case "$1" in
 			command="$(ruby -e "require 'yaml';puts (YAML.load_file('.release')['default_process_types'] || {})['$2']")"
 		fi
 		;;
-
 	*)
-		command="$@"
+		command=$*
```

Comment:
My concern with this stems from the [bash man page](http://linux.die.net/man/1/bash) on `$*`:

> Expands to the positional parameters, starting from one. **When the expansion occurs within double quotes, it expands to a single word with the value of each parameter separated by the first character of the IFS special variable**. That is, "$*" is equivalent to "$1c$2c...", where c is the first character of the value of the IFS variable. If IFS is unset, the parameters are separated by spaces. If IFS is null, the parameters are joined without intervening separators.

vs

> Expands to the positional parameters, starting from one. **When the expansion occurs within double quotes, each parameter expands to a separate word. That is, "$@" is equivalent to "$1" "$2"** ... If the double-quoted expansion occurs within a word, the expansion of the first parameter is joined with the beginning part of the original word, and the expansion of the last parameter is joined with the last part of the original word. When there are no positional parameters, "$@" and $@ expand to nothing (i.e., they are removed).

Considering that, I think we should stick to `command=$@` instead.